lib/features/create-append-elements/CreateMenuProvider.js

```diff
@@ -0,0 +1,113 @@
+import { CREATE_OPTIONS } from '../popup-menu/util/OptionsUtil';
```

Comment:
Can likely be pulled into this module, as it is only used by CreateAppendAnything as it looks like?